hal/inc/cellular_hal_cellular_global_identity.h

```diff
@@ -34,9 +34,12 @@ typedef struct __attribute__((__packed__))
 {
     uint16_t size; /*!< \c size is specified by the user application, to inform the device-os of the
                       amount of space allocated to the structure */
-    uint16_t version;  /*!< \c version is specified by the device-os, to inform the user-application
-                          of the version of the information returned */
-    uint16_t reserved; /*!< \c reserved is allocated for future usage */
+    uint16_t version; /*!< \c version is specified by the device-os, to inform the user-application
+                         of the version of the information returned */
+    uint8_t reserved; /*!< \c reserved is allocated for future usage */
+    uint8_t
```

Comment:
Can we make it a flag instead in order not to waste a whole `uint8_t` for just one boolean?

Reply:
I had considered this myself, but we have the `reserved` space, so I thought I would use it for simplicity; I'm very open to change.

Reply:
done